aliases/service_restart.yaml

```diff
@@ -6,4 +6,27 @@ name: chatops.ansible_service_restart
 action_ref: st2-chatops-aliases.service_restart
 description: Restart service on remote hosts
 formats:
-  - "service restart {{service_name}} on {{hosts}}"
+  - display: "service restart <service_name> on <hosts>"
+    representation:
+      - "service restart {{ service_name }} on {{ hosts }}"
+result:
+  format: |
+    Service restart `{{ execution.parameters.service_name }}` on `{{ execution.parameters.hosts }}` host(s): {~}
+    {% if execution.result.stderr %}
+    *Exit Status*: `{{ execution.result.return_code }}`
+    *Stderr:* ```{{ execution.result.stderr }}```
+    *Stdout:*
+    {% endif %}
+    ```{{ execution.result.stdout }}```
+  extra:
+    slack:
+      color: "{% if execution.result.succeeded %}good{% else %}danger{% endif %}"
+      fields:
+        - title: Restarted
+          value: "{{ execution.result.stdout|regex_replace('(?!SUCCESS).', '')|wordcount }}"
+          short: true
+        - title: Failed
+          value: "{{ execution.result.stdout|regex_replace('(?!(FAILED|UNREACHABLE)!).', '')|wordcount }}"
```

Comment:
This shit took half of day to figure out.
Apparently there is heavy need of more good custom Jinja filters: 
https://docs.stackstorm.com/reference/jinja_filters.html